vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -345,8 +345,7 @@ folly::dynamic ViewViewManager::GetNativeProps() const {
   auto props = Super::GetNativeProps();
 
   props.update(folly::dynamic::object("pointerEvents", "string")("onClick", "function")("onMouseEnter", "function")(
```

Comment:
>)( [](start = 63, length = 2)

(nit) could we do single tuple per line?

Reply:
I also think that would look better, but this is how clang format decided to format this.